app/src/features/surveys/observations/ObservationsMap.tsx

```diff
@@ -139,7 +139,10 @@ const ObservationsMap = () => {
               },
               {
                 layerName: 'Sample Sites',
-                features: sampleSiteFeatures.map((feature) => ({ geoJSON: feature, tooltip: <>Sample Site</> }))
+                features: sampleSiteFeatures.map((feature) => ({
+                  geoJSON: feature,
+                  tooltip: <>Sample Site</>
```

Comment:
Nit: I would probably use a `<span>` to contain this, instead of a Fragment

Reply:
updated